website/src/plugins/changelog/theme/ChangelogPage/index.tsx

```diff
@@ -5,118 +5,85 @@
+import BlogPostPageMetadata from '@theme/BlogPostPage/Metadata';
 import BlogLayout from '@theme/BlogLayout';
 import ChangelogItem from '@theme/ChangelogItem';
 import ChangelogPaginator from '@theme/ChangelogPaginator';
 import TOC from '@theme/TOC';
 import type {Props} from '@theme/BlogPostPage';
+import type {BlogSidebar} from '@docusaurus/plugin-content-blog';
 
-function ChangelogPageMetadata(props: Props): JSX.Element {
-  const {content: BlogPostContents} = props;
-  const {assets, metadata} = BlogPostContents;
-  const {title, description, date, tags, authors, frontMatter} = metadata;
-  const {keywords} = frontMatter;
-
-  const image = assets.image ?? frontMatter.image;
+function BackToIndexLink() {
+  const {metadata} = useBlogPost();
+  // @ts-expect-error: we injected this
+  const {listPageLink} = metadata;
   return (
-    <PageMetadata
-      title={title}
-      description={description}
-      keywords={keywords}
-      image={image}>
-      <meta property="og:type" content="article" />
-      <meta property="article:published_time" content={date} />
-
-      {authors.some((author) => author.url) && (
-        <meta
-          property="article:author"
-          content={authors
-            .map((author) => author.url)
-            .filter(Boolean)
-            .join(',')}
-        />
-      )}
-      {tags.length > 0 && (
-        <meta
-          property="article:tag"
-          content={tags.map((tag) => tag.label).join(',')}
-        />
-      )}
-    </PageMetadata>
+    <Link to={listPageLink}>
+      <Translate id="changelog.backLink">← Back to index page</Translate>
+    </Link>
   );
 }
 
-function ChangelogPageContent(props: Props): JSX.Element {
-  const {content: BlogPostContents, sidebar} = props;
-  const {assets, metadata} = BlogPostContents;
-  const {
-    nextItem,
-    prevItem,
-    frontMatter,
-    // @ts-expect-error: we injected this
-    listPageLink,
-  } = metadata;
+function ChangelogPageContent({
+  sidebar,
+  children,
+}: {
+  sidebar: BlogSidebar;
+  children: ReactNode;
+}): JSX.Element {
+  const {metadata, toc} = useBlogPost();
+  const {nextItem, prevItem, frontMatter} = metadata;
   const {
     hide_table_of_contents: hideTableOfContents,
     toc_min_heading_level: tocMinHeadingLevel,
     toc_max_heading_level: tocMaxHeadingLevel,
   } = frontMatter;
-
   return (
-    <>
-      <PageMetadata />
-      <BlogLayout
-        sidebar={sidebar}
-        toc={
-          !hideTableOfContents && BlogPostContents.toc.length > 0 ? (
-            <TOC
-              toc={BlogPostContents.toc}
-              minHeadingLevel={tocMinHeadingLevel}
-              maxHeadingLevel={tocMaxHeadingLevel}
-            />
-          ) : undefined
-        }>
-        <Link to={listPageLink}>
-          <Translate id="changelog.backLink">← Back to index page</Translate>
-        </Link>
+    <BlogLayout
+      sidebar={sidebar}
+      toc={
```

Comment:
remains quite duplicated with blog theme code, not sure how to dedup this

Reply:
It looks fine to me—not a big deal.